yb-voyager/src/version/constants.go

```diff
@@ -29,12 +27,22 @@ const (
 var LatestStable *YBVersion
 
 var V2024_1_3_1 *YBVersion
+var V2024_1_4_0 *YBVersion
+var V2_23_5_0 *YBVersion
 
 func init() {
 	var err error
 	V2024_1_3_1, err = NewYBVersion("2024.1.3.1")
 	if err != nil {
-		utils.ErrExit("could not create version 2024.1")
+		panic("could not create version 2024.1.3.1")
```

Comment:
can't we use utils.ErrExit?

Reply:
utils has some structs whose fields are of type `YBVersion`. so there needs to be a dependency on utils -> version. adding version->utils dependency causes circular dependency, so i thought we can just panic (As this is in the init function, it will definitely be caught in dev testing)